### PR TITLE
add parameters 31, 32

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -342,6 +342,27 @@ interface IParameterAbsoluteDataEnableMC extends IParameterAbsoluteDataEnable {
...
+ * Scan config for pulse devices.
+ * deviceParameters.PULSE_CHANNELS_SCAN_CONFIG = `31`
+ */
+interface IParameterPulseChannelsScanConfig {
+    timePullUp: number,
+    timeScan: number,
+}
+
+/**
+ * Set channels for pulse devices,
```
Comment (DarkPark) [inline src/CommandBinaryBuffer.ts:355]:
```suggestion
 * Set channels for pulse devices.
```

File: src/constants/deviceParameters.ts [hunk 2]
```diff
@@ -53,3 +53,19 @@ export const ABSOLUTE_DATA_MULTI_CHANNEL = 29;
...
+/**
+ * Scan config for pulse devices.
+ *
+ * Parameter available from SOFTWARE_VERSION = 0x6C for
+ * HARDWARE_TYPE - 15 HARDWARE_VERSION - 2 and HARDWARE_TYPE - 6 HARDWARE_VERSION - 10
+ */
+export const PULSE_CHANNELS_SCAN_CONFIG = 31;
+
+/**
+ * Set channels for pulse devices,
```
Comment (DarkPark) [inline src/constants/deviceParameters.ts:66]:
```suggestion
 * Set channels for pulse devices.
```

File: src/commands/downlink/SetParameter.ts [hunk 3]
```diff
@@ -132,6 +132,22 @@ const examples: TCommandExampleList = [
             data: {state: 1, channel: 2}
         },
         hex: {header: '03 03', body: '1e 01 01'}
+    },
+    {
+        name: 'set pulse channels config',
+        parameters: {
+            id: deviceParameters.PULSE_CHANNELS_SCAN_CONFIG,
+            data: {timePullUp: 18, timeScan: 23}
```
Comment (DarkPark) [inline src/commands/downlink/SetParameter.ts:140]: how about this change to make names more readable?

```suggestion
            data: {pullUpTime: 18, scanTime: 23}
```

File: src/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts:355]: fixed

File: src/constants/deviceParameters.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters.ts:66]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**